src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestCases/DateValue.txt

```diff
@@ -269,5 +269,8 @@ Date(1920,1,10)
 >> DateValue("1920-1-10","vi-vn")
 Date(1920,1,10)
 
->> DateValue("10 Tháng Giêng 1920","vi-vn")
-Date(1920,1,10)
\ No newline at end of file
+// Depending on .Net version, the following expression could be parsed or not
```

Comment:
>// [](http://example.com/codeflow?start=0&length=2)

I'm concerned about the loss of coverage here in the .Net Core cases. These tests were explicitly added. 

Can we move these to a separate .txt  that it excluded on 462?